build.sbt

```diff
@@ -92,21 +92,23 @@ lazy val common = project.in(file("faunadb-common"))
     )
   )
 
+crossScalaVersions := scalaVersions
+
 lazy val scala = project.in(file("faunadb-scala"))
   .dependsOn(common)
   .settings(jacoco.settings)
   .settings(publishSettings : _*)
   .settings(
     name := "faunadb-scala",
-    scalaVersion := baseScalaVersion,
+    scalaVersion := scalaVersions.last,
```

Comment:
I think this is OK but I'm a little concerned that if at some point we much with the cross-compiling versions we will inadvertently change the version of scala we're building against for main development, which may be a bit unexpected, so probably adding a comment somewhere near the `scalaVersions` definition may be useful